### ISSUE TITLE
Fix batch norm defaults

### ISSUE THREAD

**Issue by albertz** [rwth-i6/returnn_common#83]
They are wrong in RETURNN. But we can fix them here.

**Comment by albertz** [comment]
We can also fix them on RETURNN side: https://github.com/rwth-i6/returnn/issues/522

**Comment by albertz** [comment]
When we just reimplement batch norm here in returnn-common (#89), of course we also should fix them as well. The question is, what defaults to use then. Maybe the new defaults as discussed in [rwth-i6/returnn#522](https://github.com/rwth-i6/returnn/issues/522).

**Comment by albertz** [comment]
The question is also about `masked_time` (or whatever equivalent option we would have here). `masked_time=False` could have a big impact on the runtime performance efficiency, and is even the default in all other frameworks, even though it is a bit incorrect.

I would say that `masked_time` maybe should even be mandatory when there are dynamic dims, to be explicit about this behavior.

**Comment by albertz** [comment]
We wrapped the RETURNN `BatchNormLayer` now (see #89), and that one has the new defaults now (https://github.com/rwth-i6/returnn/issues/522).
